app/services/project_search_service.rb

```diff
@@ -11,8 +11,8 @@ def search(query)
     []
   end
 
-  def search_by_urn(urn)
-    Project.where("urn = ?", urn)
+  def search_by_urns(urns)
+    Project.where(urn: urns)
```

Comment:
So this has changed: are we confident we are doing enough to prevent injection here? When this is called from line 4 it is user input?

Reply:
Yes I'm confident this is OK! https://www.fastruby.io/blog/rails/security/dangerous-query-method-deprecation.html